zilliqa/Cargo.toml

```diff
@@ -43,7 +50,10 @@ clap = { version = "4.5.8", features = ["derive"] }
 crypto-bigint = "=0.5.5"
 eth_trie = {path =  "../eth-trie.rs"}
 ethabi = "18.0.0"
+ethers = { version = "2.0.14", default-features = false, features = ["ethers-solc", "legacy", "legacy-ws"] }
```

Comment:
Definitely preferred to go with alloy, except Bruno's code only uses ethers. Converting all to alloy will take some time, so I prefer to do this in stages. Like you said - it's already used in tests so temporarily I thought this could be reasonable.